src/main/scala/com/typesafe/sbt/packager/archetypes/AshScriptPlugin.scala

```diff
@@ -73,15 +73,14 @@ import SbtNativePackager.{ Universal, Debian }
  */
 object AshScriptPlugin extends AutoPlugin {
 
-  val bashTemplate = "bash-template"
-
   override def requires = JavaAppPackaging
 
-  //object autoImport extends JavaAppKeys
-
   import JavaAppPackaging.autoImport._
 
+  val ashTemplate = "ash-template"
+
   override def projectSettings = Seq(
+    bashScriptTemplateLocation := (sourceDirectory.value / "templates" / ashTemplate),
```

Comment:
I'm afraid I'm confused on this point.  I've got something like this in makeUniversalAshScript() in file  AshScriptPlugin.scala.  I've built it locally and it seems to work great for a project that consumes this feature.  Looks like this:

```scala
      val template = resolveTemplate(defaultTemplateLocation)
      val scriptBits = JavaAppAshScript.generateScript(defines, template)
      val script = tmpDir / "tmp" / "bin" / name   // fine since this is an ash-specific archetype
      IO.write(script, scriptBits)
      script.setExecutable(true)
      Some(script)
```